frontend/src/metabase/components/form/FormError/FormError.styled.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+
+import Card from "metabase/components/Card";
+import Icon from "metabase/components/Icon";
+import { alpha, color } from "metabase/lib/colors";
+
+export const ErrorCard = styled(Card)`
+  margin-top: 1.5rem;
+  margin-bottom: 2rem;
```

Comment:
Using exact value as described in https://github.com/metabase/metabase/pull/22804#discussion_r881318018